### PR TITLE
fead GAT-35 create Infinite Scroll component

### PR DESCRIPTION
**Pull request by melscastillo**

### CODE REVIEW

File: packages/gatsby-theme/package.json [hunk 1]
```diff
@@ -30,5 +31,6 @@
     "@storybook/react": "^6.1.6",
     "babel-loader": "^8.2.1",
     "eslint-import-resolver-alias": "^1.1.2"
+
```
Comment (RosaIsela31) [inline packages/gatsby-theme/package.json]:
```suggestion
```

Comment (RosaIsela31) [reply packages/gatsby-theme/package.json]: I think the line 34 is not necessary 👀

Comment (melscastillo) [reply packages/gatsby-theme/package.json]: Done! n_n

**Review by RosaIsela31 — APPROVED** [review]
LGTM! 🚀

**Review by georch — APPROVED** [review]
LGTM

**Pull request merged**